anime_downloader/util.py

```diff
@@ -221,20 +221,24 @@ def print_episodeurl(episode):
         "magnet") else episode.source().stream_url
     print(unquote(url))
 
-
-def play_episode(episode, *, player, title):
+def play_episode(episode, *, player, title, episodes="0:0"):
     if player == 'mpv':
-        p = subprocess.Popen([player, 
-            f'--title={title}', 
-            f'--referrer={episode.source().referer}', 
-            f'--user-agent={get_random_header()["user-agent"]}', 
+        p = subprocess.Popen([player,
+            f'--title={title}',
+            f'--referrer={episode.source().referer}',
+            f'--user-agent={get_random_header()["user-agent"]}',
             episode.source().stream_url])
+    elif player == "android":
+        p = subprocess.Popen(['am', 'start', '-a', 'android.intent.action.VIEW', '-t', 'video/*', '-d', f'{episode.source().stream_url}'])
+        if episodes == None or ':' in episodes:
+            if "0:1" in episodes:
+                pass
+            else:
```

Comment:
Unnecessary if-else, negate the condition by doing `if "0:1" not in episodes` instead

Reply:
Didn't think about that, lemme go fix